Dockerfile

```
@@ -0,0 +1,52 @@
+ARG PY_VERSION=3.9
+
+FROM amazon/aws-lambda-python:$PY_VERSION
+
+# Declare it a second time so it's brought into this scope.
+ARG PY_VERSION=3.9
+# Get the base output file name from the command line if provided.
+ARG FILE_NAME=example
+
+# For a list of pre-defined annotation keys and value types see:
+# https://github.com/opencontainers/image-spec/blob/master/annotations.md
+# Note: Additional labels are added by the build workflow.
+LABEL org.opencontainers.image.authors="nicholas.mcdonnell@cisa.dhs.gov"
+LABEL org.opencontainers.image.vendor="Cyber and Infrastructure Security Agency"
+
+# Bring the command line ARGs into the environment so they are available
+# in the built image.
+ENV BUILD_PY_VERSION=$PY_VERSION
+ENV BUILD_FILE_NAME=$FILE_NAME
+
+WORKDIR ${LAMBDA_TASK_ROOT}
+RUN mkdir build output
+
+# We need the zip utility to create a deployment package archive.
+RUN yum update -y \
+  && yum install -y zip \
```

Prefer long flag names:
```suggestion
RUN yum update --assumeyes \
  && yum install --assumeyes zip \
```

---

Ack, missed these when I was doing cleanup. Please see [72c616e](https://github.com/cisagov/skeleton-aws-lambda-python/pull/3/commits/72c616ebdbe0586c0911de98b647778cc832a11a).